res/css/style.css

```diff
@@ -36,6 +36,10 @@ body,
   z-index: 0;
 }
 
+#root-screenshot {
+  z-index: 1;
+}
+
```

Comment:
So, I looked at the result of this PR, and it doesn't change anything to the behavior.
The reason is simple: we want the `#root` to have a bigger `z-index` that the screenshots.
So I think this one should go to 0, and the one above to 1.

The result should be that the screenshots go _below_ the other overlays, but above the rest of the chrome.

Note: this is the current behavior:
![Capture d'écran de 2019-08-02 14-34-45](https://user-images.githubusercontent.com/454175/62370539-b4556b80-b532-11e9-835c-e3906fcb676c.png)

i hope this is clearer!

Reply:
Yep, 

> 
> 
> So, I looked at the result of this PR, and it doesn't change anything to the behavior.
> The reason is simple: we want the `#root` to have a bigger `z-index` that the screenshots.
> So I think this one should go to 0, and the one above to 1.
> 
> The result should be that the screenshots go _below_ the other overlays, but above the rest of the chrome.
> 
Alright, got it! Thanks for the great explanation!

> i hope this is clearer!
Definitely clearer!

I actually scratched my head around on how to test the changes I am doing, but I couldn't figure it out! Thanks for your help! Could you still please test me how to check the changes I am making for this one?

That way, next time - I can submit PRs in a much smarter way haha. Thanks =)